wrappers/dotnet/README.md

```diff
@@ -9,7 +9,7 @@ Pull requests welcome!
 
 ### Documentation
 
-Documentation for the .NET wrapper is available [here](http://hyperledger.github.io/indy-sdk/wrappers/dotnet/docs/index.html).
+Documentation for the .NET wrapper is available [here](https://github.com/hyperledger/indy-sdk/blob/master/wrappers/dotnet/docs/index.html).
```

Comment:
as far as I remember the initial plan was to use github.io to render .NET documentation. and there were some issues with it. Like `.io` is not enabled for HL/indy-sdk repo. And may be something else.
@srottem @tmarkovski could you share more details about this topic

Reply:
sure @jovfer . waiting for their inputs. thanks.

Reply:
Correct.  The repo needs to be configured with "Github Pages" with the source as the Master branch. See here for details: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/.

Reply:
thanks @srottem , guess maintainers/admin can do that, since i dont have access to it :-)

Reply:
Me neither.  We've been waiting for this to happen for more than a year now and it's been brought up on RocketChat multiple times, so I'm guessing no one wants to do it.

Reply:
As far as I know there are 2 main blockers:
- no one in core team has permission to enable .io
- probably there are some contradictions between this documentation and HIPE https://github.com/hyperledger/indy-hipe/tree/master/text/0025-indy-docs-framework

The first one seems just technical question for HL-desk, as I have seen some .io for others HL projects.
I will figure out how to resolve the second one.
@srottem Are you available in rocket.chat of sovrin or hyperledger for this discussion?

Reply:
thanks guys @srottem @jovfer . I am in the sovrin working group and will flag this in the call today. will update you today about the inputs i receive.